packages/@uppy/companion/src/companion.js

```diff
@@ -98,6 +98,7 @@ module.exports.app = (optionsArg = {}) => {
   }
 
   app.use(cookieParser()) // server tokens are added to cookies
+  app.get('/health', (req, res) => res.send('OK'))
```

Comment:
maybe in the future we want to send some json data here. so users should not expect an "OK" text here.
```suggestion
  app.get('/health', (req, res) => res.end())
```

Reply:
Was leaning towards just using /metrics for this Mikael, it’s a more trustworthy check, end we don’t need to introduce a second health-y endoint then

Reply:
that sounds good to me